src/main/java/no/nav/familie/integrasjoner/dokarkiv/api/ArkiverDokumentRequest.kt

```diff
@@ -6,4 +6,5 @@ import javax.validation.constraints.NotEmpty
 
 data class ArkiverDokumentRequest(@field:NotBlank val fnr: String,
                                   val isForsøkFerdigstill: Boolean,
-                                  @field:NotEmpty val dokumenter: List<Dokument>)
+                                  @field:NotEmpty val dokumenter: List<Dokument>,
+                                  val fagsakId: String?= null)
```

Comment:
We are in the process of replacing the local ArkiverDokumentRequest with the one in familie-kontrakter. It would be great if you could make this change in that project an use the class from that project instead.

Reply:
Agree that we should use the same class definition everywhere. But I think it is better to do it in another PR to replace all involved services.